### PR TITLE
Added vagrant setup for development.

### PR DESCRIPTION
**Pull request by ghost**
This change introduces vagrant setup for better development.

### CODE REVIEW

**Comment by szok** [comment]
:ok:

File: vagrant/provision.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+sudo apt-get install -y libssl-dev libxslt1-dev ncurses-dev zlib1g-dev libmemcached-dev
+sudo apt-get install -y libcap2-bin
+sudo apt-get install -y vim-nox git
+
+
+# INSTALL Ralph
+virtualenv --clear /home/vagrant
+/home/vagrant/bin/pip install -U pip
+/home/vagrant/bin/pip install -U setuptools==3.6
+virtualenv ralph
```
Comment (mkurek) [inline vagrant/provision.sh]: which virtualenv is valid? `/home/vagrant` or `ralph`?

File: vagrant/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.synced_folder "../", "/home/vagrant/sources"
```
Comment (mkurek) [inline vagrant/Vagrantfile]: (to discuss) wouldn't it be better if repo directory will be mounted under `/home/vagrant/src/ralph` (notice sources -> src and additional ralph directory)

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,14 @@
 .PHONY: test flake clean coverage doc coveralls
 
+install:
+	pip install -r requirements/base.txt
+
+install-dev:
+	pip install -r requirements/dev.txt
```
Comment (mkurek) [inline Makefile:7]: please add ipython, ipdb, pdbpp to dev requirements

File: vagrant/provision.sh [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+cd sources/
+make install-dev
+
+# CREATE db
+./vagrant/provisioning_scripts/init_mysql.sh
+
+# POST provisioning fixes
+# Add automatic activation to user .profile file
+echo 'source /home/vagrant/bin/activate' >> /home/vagrant/.profile
```
Comment (mkurek) [inline vagrant/provision.sh]: probably it isn't the only one thing which we want to add to `.profile` so could you create additional text file which then will be appended to `.profile` (for example `cat vagrant/profile_extensions >> /home/vagrant/.profile`)?

File: vagrant/provisioning_scripts/init_mysql.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "CREATE DATABASE ralph_ng DEFAULT CHARACTER SET 'utf8'" | mysql -u root
+echo "GRANT ALL ON ralph_ng.* TO ralph_ng@'%' IDENTIFIED BY 'ralph_ng'; FLUSH PRIVILEGES" | mysql -u root
+
+ralph migrate
```
Comment (vi4m) [inline vagrant/provisioning_scripts/init_mysql.sh]: /home/vagrant/bin/ralph migrate

File: Makefile (hunk 3 above)
Comment (ghost) [inline Makefile:7]: done

**Comment by mkurek** [comment]
:ok:

**Pull request merged**